projects/swarmd/swarmd/Makefile

```diff
@@ -1,15 +1,4 @@
-.PHONY: tag push
-all: push
+ORG?=linuxkitprojects
+include ../../../pkg/package.mk
```

Comment:
Actually, fixing it in this way required the `NETWORK=1` to be moved above the `include` in all the package makefiles. So rather than attaching to #2117 (which is already a bit fraught), I'll open an issue.